BenchmarkVCFs/FindSamplesAndBenchmark.wdl

```diff
@@ -0,0 +1,420 @@
+version 1.0
+import "BenchmarkVCFs.wdl" as Benchmark
+
+workflow FindSamplesAndBenchmark {
+    
+    input {
+        Array[File] input_callset
+        Array[File] ground_truth_files
+        Array[File] ground_truth_indexes
+        Array[File] ground_truth_intervals
+        Array[String] truth_labels
+        Array[File] annotation_intervals
+
+        File? gatkJarForAnnotation
+        String? annotationName
+  
+        File ref_fasta
+        File ref_fasta_index
+        File ref_fasta_dict
+        File ref_fasta_sdf
+
+        File haplotype_database 
+        File picard_cloud_jar
+
+        String docker 
+
+        File? interval_list_override
+
+        String? analysis_region
+
+        Boolean remove_symbolic_alleles=false
+
+        Array[File] stratIntervals
+        Array[String] stratLabels = [
+                         "HMER_7_and_up",
```

Comment:
since there's no default for the intervals, there shouldn't be a default for the interval labels